so/tests/imports.py

```diff
@@ -0,0 +1,2 @@
+import spt3g._libso
+from spt3g._libso import G3SuperTimestream
```

Comment:
Add this test script: https://github.com/simonsobs/so3g/blob/master/test/test_g3super.py